File: java2csharp/java/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/NotActivityException.java
package com.samsung.android.sdk.professionalaudio.widgets;

class NotActivityException extends Exception {
    /**
     * 
     */
    private static final long serialVersionUID = -6125350060493181620L;
    private static final String MSG = "Provided context is not an Activity";

    @Override
    public String getMessage() {
        return MSG;
    }
}

File: java2csharp/csharp/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/NotActivityException.cs
using System;

namespace com.samsung.android.sdk.professionalaudio.widgets
{

	internal class NotActivityException : Exception
	{
		/// 
		private const long serialVersionUID = -6125350060493181620L;
		private const string MSG = "Provided context is not an Activity";

		public override string Message
		{
			get
			{
				return MSG;
			}
		}
	}

}